### PR TITLE
Fix plots

### PR DESCRIPTION
**Pull request by jswan5**
Update code to working HTML engine; ensure compatibility with JLab

### CODE REVIEW

**Comment by misterhay** [comment]
The plots are working, which is a good start. We want to remove the code hiding and any of the "interactive questions" code cells (with their accompanying markdown cells), and simplify the wording and structure as much as possible. Any extraneous background information can be removed, as well as statements like 'Need more notes on what "predict the effects of changes in their design, alignment or composition" means.'

**Comment by jswan5** [comment]
My bad, I misunderstood the purpose of reviewing the notebooks.

Is the goal to retain the functionality and move the code to helper files, or just make the notebook overall simpler? It would be pretty straightforward to remove a lot of the wordy content, particularly in this notebook.

**Comment by misterhay** [comment]
You're right, the goal is to make the notebook simpler overall. Students usually have textbooks or other information and assessment resources, the value of Callysto is the application of coding and data science to these concepts. And we want to consider that each notebook may be the first Callysto resource that a student or educator sees.

**Comment by jswan5** [comment]
Thanks for the clarification @misterhay. I made some major changes to the notebook, though I'm on the fence about the final section with Total Internal Reflection, it seems like a big jump in complexity without much supporting info. I've left it in for now, but I'd be curious to get feedback from an education specialist on keeping it in.

**Comment by misterhay** [comment]
I've made some updates to wording, deleted extra image files, and simplified the code slightly. @meghbali could you have a look and add a review?

**Comment by meghbali** [comment]
I have started reviewing ...

**Comment by meghbali** [comment]
Didn't we want to move the codes to separate py files? just checking.

**Review by meghbali — APPROVED** [review]
